README.MD

```diff
@@ -103,9 +104,22 @@ export class UsersController extends Controller {
   }
 }
 ```
+
 Note: tsoa can not create swagger documents from interfaces that are defined in external dependencies. This is by design. Full explanation available in [ExternalInterfacesExplanation.MD](https://github.com/lukeautry/tsoa/blob/master/docs/ExternalInterfacesExplanation.MD)
 
+### Mono Repos
```

Comment:
This should be part of the External Interfaces Explanation.

Reply:
@WoH should I update the external interfaces only or do you want information about this in the base readme as well?

Reply:
Mostly the external interfaces, I added a small change to the base readme as a suggestion/inspiration because I think the language was not very clear.